### PR TITLE
Dịch "dcgan_vn" - Phần 3

### LINKED ISSUE #2739: Dịch "dcgan_vn" - Phần 3
Phần cần dịch: https://github.com/aivivn/d2l-vn/blame/master/chapter_generative-adversarial-networks/dcgan_vn.md#L468-L609

### Hướng dẫn
- [Dịch trực tiếp trên website](https://github.com/aivivn/d2l-vn/blob/master/CONTRIBUTING.md).
- Sau khi nhấn vào phần cần dịch ở trên, hãy chọn **Normal view** và làm theo hướng dẫn ở trên để có thể bắt đầu dịch.

### PR DESCRIPTION
**Pull request by trGiang99**
## Pull Request Checklist

**Đánh dấu tick (sau khi "Create Pull Request") vào các mục dưới đây:**

* [x] Pull Request này tương ứng với issue nào? Điền số issue sau dấu `#` (không có dấu cách): Close #2739

* [x] Bản dịch này có bám sát **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)** không? Nếu một từ/cụm chưa có trong Bảng thuật ngữ, hãy thảo luận thêm ở thẻ Discussions của dự án.

* [x] Format của bản gốc có được giữ nguyên không?
     
* [x] Trong một đoạn văn, mỗi câu văn đã được viết trong một dòng, giữa các dòng không có dòng trắng. Markdown sẽ coi những dòng liền nhau không có dòng trắng là một đoạn văn. Việc này giúp công đoạn review được thuận tiện hơn. Bạn đã kiểm tra rằng mỗi đoạn văn đã nằm một dòng?

* [x] Điền tên đầy đủ của bạn vào mục **Những người thực hiện** ở cuối tập tin.

## Lưu ý cho người phản biện (Reviewer)

1. Chỉ review khi Pull Request (PR) này đã sẵn sàng cho việc review. Một PR sẵn sàng khi PR đó không có label `status: wip`.
2. Đã đọc kỹ hướng dẫn đóng góp và xem qua **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)**.

### CODE REVIEW

File: chapter_generative-adversarial-networks/dcgan_vn.md [hunk 1]
```diff
@@ -593,7 +599,10 @@ train(net_D, net_G, data_iter, num_epochs, lr, latent_dim)
 It aims to fix the “dying ReLU” problem and helps the gradients flow easier through the architecture.
 -->
 
-*dịch đoạn phía trên*
+* Kiến trúc DCGAN gồm có bốn tầng tích chập cho Bộ phân biệt và bốn tầng tích chập "sải bước một phần (*fractionally-strided*)" cho Bộ sinh.
+* Bộ phân biệt là một mạng tích chập 4 tầng có sải bước với chuẩn hoá theo batch (trừ tầng đầu vào) và hàm kích hoạt ReLU rò rỉ.
```
Comment (rootonchair) [inline chapter_generative-adversarial-networks/dcgan_vn.md:603]:
```suggestion
* Bộ phân biệt là một mạng 4 tầng bao gồm các tầng tích chập có sải bước và chuẩn hoá theo batch (trừ tầng đầu vào) và hàm kích hoạt ReLU rò rỉ.
```

File: chapter_generative-adversarial-networks/dcgan_vn.md [hunk 2]
```diff
@@ -593,7 +599,10 @@ train(net_D, net_G, data_iter, num_epochs, lr, latent_dim)
...
+* ReLU rò rỉ là một hàm phi tuyến trả về kết quả khác không với đầu vào âm.
+Hàm này nhằm khắc phục vần đề "chết ReLU" và giúp gradient truyền một cách dễ dàng hơn trong suốt kiến trúc.
```
Comment (rootonchair) [inline chapter_generative-adversarial-networks/dcgan_vn.md:605]:
```suggestion
Hàm này nhằm khắc phục vần đề "ReLU chết" và giúp gradient truyền một cách dễ dàng hơn trong suốt kiến trúc.
```

File: chapter_generative-adversarial-networks/dcgan_vn.md [hunk 3]
```diff
@@ -603,7 +612,8 @@ It aims to fix the “dying ReLU” problem and helps the gradients flow easier
 2. Apply DCGAN on Fashion-MNIST and see which category works well and which does not.
 -->
 
-*dịch đoạn phía trên*
+1. Chuyện gì sẽ xảy ra nếu ta sử dụng hàm kích hoạt ReLU phổ thông thay phì ReLU rò rỉ?
```
Comment (rootonchair) [inline chapter_generative-adversarial-networks/dcgan_vn.md:615]:
```suggestion
1. Chuyện gì sẽ xảy ra nếu ta sử dụng hàm kích hoạt ReLU phổ thông thay vì ReLU rò rỉ?
```

**Review by rootonchair — APPROVED** [review]
LGTM

File: chapter_generative-adversarial-networks/dcgan_vn.md [hunk 4]
```diff
@@ -483,7 +483,12 @@ to take care of the rapid changing gradients because the generator and the discr
 Besides, the random generated noise `Z`, is a 4-D tensor and we are using GPU to accelerate the computation.
 -->
 
-*dịch đoạn phía trên*
+So với mô hình GAN cơ bản trong :numref:`sec_basic_gan`,
+ta sử dụng cùng tốc độ học cho cả bộ sinh và bộ phân biệt do chúng tương đồng với nhau.
+Thêm nữa, ta thay đổi $\beta_1$ trong Adam (:numref:`sec_adam`) từ $0.9$ về $0.5$.
+Việc này làm giảm độ mượt của động lượng, tức là trung bình động trọng số mũ của các gradient trước đó,
+nhằm đáp ứng sự thay đổi nhanh chóng của grandient do bộ sinh và bộ phân biệt đối kháng lẫn nhau.
```
Comment (minhduc0711) [inline chapter_generative-adversarial-networks/dcgan_vn.md:490]:
```suggestion
nhằm đáp ứng sự thay đổi nhanh chóng của gradient do bộ sinh và bộ phân biệt đối kháng lẫn nhau.
```

File: chapter_generative-adversarial-networks/dcgan_vn.md [hunk 5]
```diff
@@ -575,7 +580,8 @@ We train the model with a small number of epochs just for demonstration.
 For better performance, the variable `num_epochs` can be set to a larger number.
 -->
 
-*dịch đoạn phía trên*
+Chúng tôi chỉ huấn luyện mô hình với số epoch nhỏ để minh hoạ.
+Để đạt hiệu năng tốt hơn, bạn có thể đặt biến `num_epochs` bằng một số lớn hơn.
```
Comment (minhduc0711) [inline chapter_generative-adversarial-networks/dcgan_vn.md:584]:
```suggestion
Để đạt chất lượng mô hình tốt hơn, bạn có thể đặt biến `num_epochs` bằng một số lớn hơn.
```

File: chapter_generative-adversarial-networks/dcgan_vn.md [hunk 6]
```diff
@@ -593,7 +599,10 @@ train(net_D, net_G, data_iter, num_epochs, lr, latent_dim)
 It aims to fix the “dying ReLU” problem and helps the gradients flow easier through the architecture.
 -->
 
-*dịch đoạn phía trên*
+* Kiến trúc DCGAN gồm có bốn tầng tích chập cho Bộ phân biệt và bốn tầng tích chập "sải bước một phần (*fractionally-strided*)" cho Bộ sinh.
+* Bộ phân biệt là một mạng 4 tầng bao gồm các tầng tích chập có sải bước và chuẩn hoá theo batch (trừ tầng đầu vào) và hàm kích hoạt ReLU rò rỉ.
+* ReLU rò rỉ là một hàm phi tuyến trả về kết quả khác không với đầu vào âm.
+Hàm này nhằm khắc phục vần đề "ReLU chết" và giúp gradient truyền một cách dễ dàng hơn trong suốt kiến trúc.
```
Comment (minhduc0711) [inline chapter_generative-adversarial-networks/dcgan_vn.md:605]:
```suggestion
Hàm này nhằm khắc phục vần đề "ReLU chết" và giúp gradient truyền một cách dễ dàng hơn xuyên suốt kiến trúc.
```

**Review by minhduc0711 — APPROVED** [review]
thanks, mình thêm vài cái nhỏ thôi

File: chapter_generative-adversarial-networks/dcgan_vn.md [hunk 7]
```diff
@@ -593,7 +599,10 @@ train(net_D, net_G, data_iter, num_epochs, lr, latent_dim)
 It aims to fix the “dying ReLU” problem and helps the gradients flow easier through the architecture.
 -->
 
-*dịch đoạn phía trên*
+* Kiến trúc DCGAN gồm có bốn tầng tích chập cho Bộ phân biệt và bốn tầng tích chập "sải bước một phần (*fractionally-strided*)" cho Bộ sinh.
```
Comment (minhduc0711) [inline chapter_generative-adversarial-networks/dcgan_vn.md:602]: À fyi từ `fractionally-strided` trước mình thấy có ai đó dịch là `sải bước phân số` thì phải. 
Bản thân mình cũng không hiểu rõ nguồn gốc của cái tên này lắm nên cũng k biết dịch thế nào đúng hơn.

Comment (trGiang99) [reply chapter_generative-adversarial-networks/dcgan_vn.md:602]: Qua phần code thì có vẻ như 4 tầng tích chập có số stride khác nhau nên được gọi như vậy. Chỗ này em cũng không rõ lắm.

**Review by duythanhvn — APPROVED** [review]

File: chapter_generative-adversarial-networks/dcgan_vn.md (hunk 7 above)
Comment (minhduc0711) [reply chapter_generative-adversarial-networks/dcgan_vn.md:602]: nó là tên gọi khác cho `transposed convolution` thôi, chắc ngụ ý việc tầng này phóng to feature map với "stride < 1"

Comment (trGiang99) [reply chapter_generative-adversarial-networks/dcgan_vn.md:602]: Em cũng mới đọc qua, đúng là tên gọi này xuất phát từ việc coi `transposed convolution` tương đương với cnn có s < 1.
`sải bước phân đoạn` thì sao nhỉ?

Comment (rootonchair) [reply chapter_generative-adversarial-networks/dcgan_vn.md:602]: nếu vậy thì sẽ là `segmental stride`, không phù hợp.  Mình thấy `một phần` mang đủ nghĩa

Comment (minhduc0711) [reply chapter_generative-adversarial-networks/dcgan_vn.md:602]: để mình bê qua discussion #2847

**Comment by duythanhvn** [comment]
Anh merge điểm này trước, phần thay đổi thuật ngữ ở #2847 sẽ update trong phần revise nhé.
